catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/event/SendEvent.java

```diff
@@ -234,28 +271,31 @@ public void deleted(Metacard oldMetacard) {
 
     }
 
+    private long introduceJitter(long value, double percent) {
+        return new Random().nextLong() % (Math.round(value * percent));
+    }
+
     @Override
     public boolean isAvailable() {
-        if (hasExpiration(subject)) {
-            if (!Security.getInstance()
-                    .tokenAboutToExpire(subject)) {
+        if (retryCount > 0) {
+            long retryTimeOffset = (long) Math.pow(2, Math.min(retryCount, MAX_RETRY_COUNT)) * 50;
```

Comment:
Might be worth putting in a comment indicating what the range of offsets values are so no one feels compelled to do the math.